orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -495,8 +514,12 @@ def testInstantiateORTTrainer(step_fn):
         output = trainer.eval_step(data, targets)
     elif step_fn == 'train_step':
         step_fn = trainer.train_step
-        data, targets = utils.get_batch(train_data, 0)
-        output = trainer.train_step(data, targets)
+        for i in range(max_train_step):
+            data, targets = utils.get_batch(train_data, 0)
+            output = trainer.train_step(data, targets)
```

Comment:
If `output`is not being used, you can replace it by `_` to make it clear that it is not important

Reply:
it is used to check that the dimensions of output are correct. Line 528